subsys/caf/modules/buttons.c

```diff
@@ -242,7 +282,7 @@ static void scan_fn(struct k_work *work)
 	}
 
 	/* Avoid draining current between scans */
-	if (set_cols(0x00000000)) {
+	if (set_cols(MASK_NONE, true)) {
```

Comment:
Renamed to `enforce_pin_state` - output enforces pin state (i.e. does not allow for floating pings)

Reply:
I see now where true comes from. Since I could not get it the first time I think the name really is misleading.

I still think it should be false, as argument is ignored. Going back to the previous comment. I don't think enforce_pin_state is correct - you cannot enforce pin state with pull. The state will be known only when something else won't drive the pin. The name is so misleading - it would be easier to follow if argument says what it really is, a pull configuration when pin is an input.